### PR TITLE
Select shoppers activity

### PR DESCRIPTION
**Pull request by assafgelber**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -8,7 +9,15 @@
         android:label="@string/app_name"
         android:theme="@style/AppTheme" >
         <activity
-            android:name=".ui.request_flow.RequestStateActivity"
+            android:name=".ui.request_flow.Activities.SelectShopperActivity"
+            android:label="@string/title_activity_select_shopper" >
+          <intent-filter>
+            <action android:name="android.intent.action.MAIN"/>
```
Comment (AsafGartner) [inline app/src/main/AndroidManifest.xml]: Make sure you remove these before the merge.
Only the `HomeActivity` should have MAIN/LAUNCHER.

File: app/src/main/java/com/iic/shopingo/ui/request_flow/Activities/RequestStateActivity.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.iic.shopingo.ui.request_flow;
+package com.iic.shopingo.ui.request_flow.Activities;
```
Comment (ifeins) [inline app/src/main/java/com/iic/shopingo/ui/request_flow/Activities/RequestStateActivity.java]: Don't name packages with uppercase letters, that causes issues between different OS.

File: app/src/main/java/com/iic/shopingo/ui/request_flow/Activities/SelectShopperActivity.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+import com.iic.shopingo.R;
+import com.iic.shopingo.ui.request_flow.Views.SelectShopperListItemView;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SelectShopperActivity extends ActionBarActivity {
+  @InjectView(R.id.select_shopper_list)
+  ListView mShopperList;
+
+  private SelectShopperAdapter mAdapter;
```
Comment (ifeins) [inline app/src/main/java/com/iic/shopingo/ui/request_flow/Activities/SelectShopperActivity.java]: Let's ditch the "m" prefix from variable names, it's mostly a C++ convention (even though the AOSP adopted it).
But from looking in open source libraries for android such as retrofit, butterknife, ORMLite, I see that none of them use this "m" prefix convention.

Also it makes it harder to generate getters/setters (i.e. if you have a member called mBlat then it generates getmBlat() method).

**Comment by ifeins** [comment]
:+1:

**Pull request merged**